tasks_management/services.py

```diff
@@ -275,6 +285,9 @@ def _adjust_delete_task_data(self, obj_data):
                 obj_data[key] = str(obj_data[key])
         return obj_data
 
+    def _data_for_json_ext_delete(self, obj_data):
```

Comment:
Same as previous raised comments with 'context' and potential redundant 'argument'